pkg/sfu/buffer/rtpstats.go

```diff
@@ -87,6 +89,11 @@ type RTCPSenderReportData struct {
 	ArrivalTime  time.Time
 }
 
+type RTCPSenderReportInfo struct {
```

Comment:
naming-wise, this is a bit hard to know what's the difference is between `*Info` vs `*Data`. Does it make sense to either
* Add `SmoothedOWD` to RTCPSenderReportData
* Rename this struct to `RTCPSenderReportDataExt`

Reply:
Will rename to `RTCPSenderReportDataExt`. Reason I did not include it in `RTCPSenderReportData` is because that goes across the relay protocol and also used when setting that data when a RTCP sender report is received. As the smoothed owd is a calculated quantity, felt odd to have an extra field in there 😄

Reply:
Renaming in this [commit](https://github.com/livekit/livekit/pull/1321/commits/3f03ad7dd8b355e2b2bc957bf9c91a928e088f97).